packages/components/pages/f-checkout/src/components/Notes.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+    <div v-if="isSplitNotesEnabled">
+        <accordion
+            :id="noteTypeCourierOrOrder"
+            :title="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.text`) }}</span>
+            <form-field
+                :placeholder="$t(`userNote.${noteTypeCourierOrOrder}.${serviceType}.placeholder`)"
+                :value="noteValue"
+                v-bind="inputStyles"
+                :name="`${noteTypeCourierOrOrder}-note`"
+                @input="updateUserNotes({ note: $event, type: noteTypeCourierOrOrder })" />
+        </accordion>
+        <accordion
+            v-if="shouldShowKitchenNotes"
+            id="kitchen"
+            :title="$t(`userNote.kitchen.${serviceType}.title`)">
+            <span :class="$style['c-checkout-accordion-help']">{{ $t(`userNote.kitchen.${serviceType}.text`) }}</span>
+            <form-field
+                :placeholder="$t(`userNote.kitchen.${serviceType}.placeholder`)"
+                :value="kitchenNoteValue"
+                v-bind="inputStyles"
+                name="kitchen-note"
+                @input="updateUserNotes({ note: $event, type: 'kitchen' })" />
+        </accordion>
+    </div>
+    <form-field
+        v-else-if="!isSplitNotesEnabled"
+        :label-text="$t(`userNote.order.delivery.title`)"
+        :placeholder="$t(`userNote.order.${serviceType}.placeholder`)"
+        :value="noteValue"
+        v-bind="inputStyles"
+        :name="'order-note'"
+        @input="updateUserNotes({ note: $event, type: 'order' })" />
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex';
+import FormField from '@justeat/f-form-field';
+import Accordion from './Accordion.vue';
+import '@justeat/f-form-field/dist/f-form-field.css';
```

Comment:
Nit: I think it's good to keep the external and internal imports separate
```suggestion
import FormField from '@justeat/f-form-field';
import '@justeat/f-form-field/dist/f-form-field.css';

import Accordion from './Accordion.vue';
```

Reply:
Agreed. Oversight on my part.